ui.html

```diff
@@ -76,4 +84,80 @@
       parent.postMessage({ pluginMessage: { type: 'helpoff' } }, '*')
     }
   }
+
+
+  /** 
+  * message events coming from code.ts where the main logic for the timer sits
+  * messages contain timer information by ID, so that we can update UI.html with timer info
+  */
+
+  onmessage = (event) => {
+
+    var progressBarWidth = 200;
+    var progressBarHeight = 10;
+    var progressBarMargin = 5;
+
+    var eventType = event.data.pluginMessage[0];
+    var timerText = event.data.pluginMessage[1];
+    var timerID = event.data.pluginMessage[2];
+    var secondsToGo = event.data.pluginMessage[3];
+    var secondsToStart = event.data.pluginMessage[4];
+
+    if (eventType == "start timer") {
```

Comment:
We can convert the if statements into switch case

Reply:
done